### DISCUSSION TITLE
Allow to use secret in non yaml mode?

### DISCUSSION THREAD

**Discussion by timgreen** [home-assistant/frontend#14998] [Dashboard] [answered]
As mentioned in https://community.home-assistant.io/t/solved-secret-and-lovelace/94060/2, now the only workaround is to switch to whole dashboard to yaml mode.

**Comment by frenck** [comment, marked as answer]
That is not a frontend feature, by mostly a backend feature. The community forums are, therefore, the right place to request this specific feature.

Secrets are mainly aimed at the backend and were originally created for sharing basic YAML configuration via, e.g., GitHub. This does not apply in this case, as these UI-managed dashboards are stored elsewhere.

Anyways, the forums are the right place for this specific request/discussion.

Thanks!

../Frenck
